sdks/go/pkg/beam/core/runtime/exec/fn_arity.tmpl

```diff
@@ -49,22 +51,19 @@ func (n *invoker) initCall() {
 
             // (5) Return direct output, if any. Input timestamp and windows are implicitly
```

Comment:
This file has a weird mix of tabs and spaces.

Reply:
Done.